### DISCUSSION TITLE
Wow! Being Invited by GitHub Recruitment Team in the comments of  other's personal blog!

### DISCUSSION THREAD

**Discussion by sify21** [community/community#109411] [Code Security] [answered]
### Select Topic Area

Product Feedback

### Body

https://github.com/linyuliang/linyuliang.github.io/pull/7#issuecomment-1955447284

Is there a setting on github to stop being notified when someone `@` me in a thread or repository that I didn't subscribe, watch or participate in?

**Comment by ettaboyle** [comment, marked as answer]
We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.
